curriculum/challenges/english/25-front-end-development/lecture-working-with-images-and-svgs/67167835def3588873f339c6.md

```diff
@@ -74,23 +67,23 @@ PNG
 
 ### --feedback--
 
-Review the video starting at the 53 second mark to see which of these options was not mentioned.
+Review the beginning of the lecture to see which of these options was not mentioned.
```

Comment:
it is not the beginning of the lecture tho, of the four paragraphs, it is the third one